tests/integration/main.rs

```diff
@@ -0,0 +1,93 @@
+mod artichoke;
+
+use std::env;
+use std::path::PathBuf;
+use std::process::Command;
+use serde::ser::{Serialize, Serializer, SerializeStruct};
```

Comment:
If the fmt job is finicky, I think it might ask for an empty line between `std` and 3p import groups.

`bundle exec rake fmt` will run nightly rustfmt which should fix this up.